### PR TITLE
feat(node): add Generic OS Debian (#549)

### PR DESCRIPTION
**Pull request by gabriel-tessier**
PR for this issue just added Debian icon and run autogen:
https://github.com/mingrammer/diagrams/issues/549

### CODE REVIEW

**Review by plaes — COMMENTED** [review]
This pull request contains bunch of other irrelevant changes..

File: diagrams/ibm/network.py [hunk 1]
```diff
@@ -93,19 +93,3 @@ class VpnPolicy(_Network):
...
-FW = Firewall
-LB = LoadBalancer
-NLB = LoadBalancer 
-PublicGateway = Gateway
-SecurityGroupRules = Rules
-TG = TransitGateway
-VPC = Vpc
-VPNGateway = VpnGateway
-VPNConnection = VpnConnection
-VPNPolicy = VpnPolicy
```
Comment (mingrammer) [inline diagrams/ibm/network.py:111]: Oh.. we may lose the alias map somewhere. We should recover it.

**Comment by gabriel-tessier** [comment]
@mingrammer 
I merged with master, the irrelevant changes pointed by @plaes are gone and only my changes was kept.
I also resize the picture, the first version was 512px.

I hope that the last changes are correct.

**Review by mingrammer — APPROVED** [review]
LGTM!

**Pull request merged**